nextstrain/cli/runner/conda.py

```diff
@@ -194,27 +222,23 @@ def setup_micromamba(dry_run: bool = False, force: bool = False) -> bool:
         if not dry_run:
             shutil.rmtree(str(MICROMAMBA_ROOT))
 
-    # Query for Micromamba release
     try:
-        dist = package_distribution("conda-forge", "micromamba", MICROMAMBA_VERSION)
+        subdir = platform_subdir()
     except InternalError as err:
         warn(err)
         return False
 
-    assert dist, f"unable to find micromamba dist"
-
-    # download_url is scheme-less, so add our preferred scheme but in a way
-    # that won't break if it starts including a scheme later.
-    dist_url = urljoin("https:", dist["download_url"])
+    url = urljoin(CHANNEL_ALIAS, MICROMAMBA_URL.replace('{subdir}', subdir))
+    assert url, f"unable to find micromamba url"
 
-    print(f"Requesting Micromamba from {dist_url}…")
+    print(f"Requesting Micromamba from {url}…")
 
     if not dry_run:
-        response = requests.get(dist_url, stream = True)
+        response = requests.get(url, stream = True)
         response.raise_for_status()
         content_type = response.headers["Content-Type"]
 
-        assert content_type == "application/x-tar", \
+        assert content_type in {"application/x-tar", "binary/octet-stream"}, \
```

Comment:
Now that `NEXTSTRAIN_CONDA_MICROMAMBA_URL` can be any URL, is there a chance that the URL could point to a `binary/octet-stream` that doesn't work well with `tarfile.open` and end up raising a `tarfile.ReadError`?

Reply:
> or maybe just stop checking types all together?

I did this and just handle the error instead.